metau-capstone/ScheduleViewController.m

```diff
@@ -0,0 +1,44 @@
+//
+//  ScheduleViewController.m
+//  metau-capstone
+//
+//  Created by Shirley Zhu on 7/6/22.
+//
+
+#import "ScheduleViewController.h"
+#import "Parse/Parse.h"
+#import "SceneDelegate.h"
+#import "LoginViewController.h"
+
+@interface ScheduleViewController ()
+
+@end
+
+@implementation ScheduleViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+- (IBAction)didTapLogout:(id)sender {
+    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
```

Comment:
can you create a shared utility file with shared functions for hosting the common functionality since logout button on all 3 tabs perform the same action? DRY (Don't Repeat Yourself)